pkgdb/tests/setup_suite.bash

```diff
@@ -83,14 +83,37 @@ misc_vars_setup() {
 
   export _PKGDB_TEST_SUITE_MODE=:
 
-  NIXPKGS_REV="e8039594435c68eb4f780f3e9bf3972a7399c4b1"
+  # Incomplete notes on steps needed to bump:
+  # _PKGDB_GA_REGISTRY_REF_OR_REV=$NIXPKGS_REV \
+  #   pkgdb manifest lock --ga-registry --manifest pkgdb/tests/harnesses/proj1/manifest.toml \
+  #   | jq > pkgdb/tests/harnesses/proj1/manifest.lock
+  NIXPKGS_REV="ab5fd150146dcfe41fda501134e6503932cc8dfd"
   NIXPKGS_REF="github:NixOS/nixpkgs/$NIXPKGS_REV"
 
-  NIXPKGS_FINGERPRINT="5fde12e3424840cc2752dae09751b09b03f5a33"
-  NIXPKGS_FINGERPRINT="${NIXPKGS_FINGERPRINT}c3ec4de672fc89d236720bdc7"
+  NIXPKGS_FINGERPRINT="9bb3d4c033fbad8efb5e28ffcd1d70383e0c5bb"
+  NIXPKGS_FINGERPRINT="${NIXPKGS_FINGERPRINT}cb7cc5c526b824524467b19b9"
```

Comment:
suggestion: maybe a short comment how the fingerprints are generated. 
-- still look kinda arbitrary